[D1] Add wrangler d1 insights (#13547)
cloudflare/cloudflare-docs: content/d1/observability/metrics-analytics.md

The metrics displayed in the [Cloudflare dashboard](https://dash.cloudflare.com/) charts are queried from Cloudflare’s [GraphQL Analytics API](/analytics/graphql-api/). You can access the metrics [programmatically](#query-via-the-graphql-api) via GraphQL or HTTP client.

> ## View via the dashboard

Reviewer: ```suggestion
## View metrics in the dashboard
```

	}
}
```

## Query insights
D1 exposes metrics that let you understand and debug query performance. These metrics can be accessed via GraphQL's `d1QueriesAdaptiveGroups` or `wrangler d1 insights`command.

> - Query strings are captured without bound [parameters](/d1/build-with-d1/d1-client-api/#parameter-binding), which can contain sensitive information

Reviewer 2: ```suggestion
- Query strings are captured without bound [parameters](/d1/build-with-d1/d1-client-api/#parameter-binding), which can contain sensitive information.
```

> - Query strings are captured without bound [parameters](/d1/build-with-d1/d1-client-api/#parameter-binding), which can contain sensitive information

Reviewer: ```suggestion
- Query strings are captured without bound [parameters](/d1/build-with-d1/d1-client-api/#parameter-binding), to make it easier to analyze metrics across queries.
```

> - Experimental wrangler command options and output can change, run `wrangler d1 insights --help` to view current command

Reviewer 2: ```suggestion
- Experimental wrangler command options and output can change. Run `wrangler d1 insights --help` to view the current command.
```
